app/src/ui-blokada/kotlin/core/bits/Slots.kt

```diff
@@ -68,14 +68,21 @@ class FiltersStatusVB(
         Unit
     }
 
+    fun getMaxMemory(): Int {
+        val runtime = Runtime.getRuntime()
+        val totalmem = runtime.maxMemory()
+        val mb = totalmem.div(1048576)
```

Comment:
What is this magic number?

Reply:
The return value is bytes, to display it in MB, I have to divide the result 2 times with 1024. That's the magic number.